src/ekobox/evol_pdf.py

```diff
@@ -100,28 +90,5 @@ def evolve_pdfs(
         genpdf.install_pdf(name)
 
 
-def gen_out(theory_card, op_card, path=None):
-    """
-    Generates EKO output from theory and operators cards and, if requested,
-    dumps it in tar format
-
-    Parameters
-    ----------
-        theory_card : dict
-            theory card
-        op_card : dict
-            operators card
-        path : str
-            path of dumped output (if "None" output is not dumped)
-
-    Returns
-    -------
-        : eko.output.Output
-            eko output
-    """
-    eko_output = eko.run_dglap(theory_card, op_card)
-    if path is not None:
-        ops_id = f"o{op_card['hash'][:6]}_t{theory_card['hash'][:6]}"
-        path = f"{path}/{ops_id}.tar"
-        eko.output.legacy.dump_tar(eko_output, path)
-    return eko_output
+def ekofileid(theory_card, operators_card):
```

Comment:
Missing docstring

Reply:
agreed - still I hope that function will soon vanish completely